homeassistant/components/vera/common.py

```diff
@@ -25,12 +26,12 @@ class ControllerData(NamedTuple):
 
 def get_configured_platforms(controller_data: ControllerData) -> set[str]:
     """Get configured platforms for a controller."""
-    platforms = []
+    platforms: Sequence[str] = []
     for platform in controller_data.devices:
         platforms.append(platform)
 
     if controller_data.scenes:
-        platforms.append(SCENE_DOMAIN)
+        platforms.append(Platform.SCENE)
```

Comment:
We should update the values in `type_map` in the `map_vera_device` function in the init module too.
https://github.com/home-assistant/core/blob/c8f2d4a82bcaab59bfb453793d4246c9e7e105bc/homeassistant/components/vera/__init__.py#L187-L197

As well as this line:
https://github.com/home-assistant/core/blob/c8f2d4a82bcaab59bfb453793d4246c9e7e105bc/homeassistant/components/vera/__init__.py#L201